monitoring/config/alert.rules.tmpl

```diff
@@ -0,0 +1,80 @@
+groups:
+- name: High CPU
+  rules:
+%{ for app in apps ~}
+  - alert: High-CPU-${app}
+    expr: avg by ( app ) (cpu{app=~"${app}"}) > 50
+    for: 5m
+    annotations:
+      summary:     ${app} High CPU Alert
+      dashboard:   ${grafana_dashboard_url}&var-Applications=${app}
+      description: "CPU usage of at least 50% for more than 5 minutes (current value: {{ $value }})"
+    labels:
+      environment: production
+      severity:    high
+      app:         ${app}
+%{ endfor ~}
+
+- name: Memory Utilisation
+  rules:
+%{ for app in apps ~}  
+  - alert: High-Memory-Utilisation-${app}
+    expr: avg by ( app ) (memory_utilization{app=~"${app}"}) > 60
+    for: 5m
+    annotations:
+      summary:     ${app} high memory utilization
+      dashboard:   ${grafana_dashboard_url}&var-Applications=${app}
+      description: "Memory utilization at least 60% for more than 5 minutes (current value: {{ $value }})"
+    labels:
+      severity:    high
+      app:         ${app}
+      environment: production
+%{ endfor ~}
+
+- name: Disk Utilisation
+  rules:
+%{ for app in apps ~}  
+  - alert: High-Disk-Utilisation-${app}
+    expr: avg by ( app ) ( disk_utilization{ app=~"${app}" }) > 60
+    for: 5m
+    annotations:
+      summary:     ${app} high disk utilization
+      dashboard:   ${grafana_dashboard_url}&var-Applications=${app}
+      description: "Disk utilization is at least 60% for more than 5 minutes (current value: {{ $value }})"
```

Comment:
Maybe remove the hard coded 60% here so if we change later we only change in one place?
for example: "High disk usage for more than 5 minutes (current value: {{ $value }})"